jans-auth-server/model/src/main/java/io/jans/as/model/jwe/Jwe.java

```diff
@@ -49,6 +49,10 @@ public static Jwe parse(String encodedJwe, PrivateKey privateKey, byte[] sharedS
         return jwe;
     }
 
+    public Jwe encryptJwe(Jwe jwe, JweEncrypter jweEncrypter) throws InvalidJweException {
```

Comment:
Lets drop this method. Not clear why do we need it here. If we need higher level of abstraction for encryption then lets add it. Calling jwe drom jweEncrypter or jweEncrypted from jwe does not solve anything.